core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/statistics/percentile.kt

```diff
@@ -0,0 +1,34 @@
+package org.jetbrains.kotlinx.dataframe.statistics
+
+import io.kotest.matchers.shouldBe
+import org.jetbrains.kotlinx.dataframe.api.Infer
+import org.jetbrains.kotlinx.dataframe.api.columnOf
+import org.jetbrains.kotlinx.dataframe.api.dataFrameOf
+import org.jetbrains.kotlinx.dataframe.api.mapToColumn
+import org.jetbrains.kotlinx.dataframe.api.percentile
+import org.jetbrains.kotlinx.dataframe.api.rowPercentile
+import org.junit.Test
+
+@Suppress("ktlint:standard:argument-list-wrapping")
+class PercentileTests {
+
+    @Test
+    fun `percentile of two columns`() {
+        val df = dataFrameOf("a", "b")(
+            1, 4,
+            2, 6,
+            7, 7,
+        )
+        df.percentile(60.0, "a", "b") shouldBe 6
+    }
+
+    @Test
+    fun `row percentile`() {
+        val df = dataFrameOf("a", "b", "c")(
+            1, 3, 3,
+            2, 4, 10,
+            7, 7, 1,
+        )
+        df.mapToColumn("", Infer.Type) { it.rowPercentile(25.0) } shouldBe columnOf(1, 2, 1)
```

Comment:
Btw. Do you think it would be a good idea to offer an enum? Like `Percentile.Q1`, `Percentile.Q2`, etc.

Reply:
or probably an `object` with `val Q1 = 0.25`, etc.

Reply:
Well, it could be a good idea, but for statistics library :). I believe simple percentile is enough for now.

Reply:
true